app/models.py

```diff
@@ -12,7 +14,18 @@ class Plant(db.Model):
     root_depth = db.Column(db.Integer)
     annual = db.Column(db.Boolean)
 
+    def in_garden(self):
+        Garden.query.filter_by(plant_id=self.id).first()
+
     def random_id():
         from random import randint
         num = randint(1, 24)
         return num
+
+class Garden(db.Model):
+    __tablename__ = 'gardens'
+    id = db.Column(db.Integer, primary_key=True)
+    plant_id = db.Column(db.Integer, db.ForeignKey('plants.id'))
+    plant_name = db.Column(db.String(60))
+    last_watered = db.Column(db.DateTime)
+    date_added = db.Column(db.DateTime, default=datetime.now())
```

Comment:
Nice work investigating datetime.now.  I like that it auto-fills when the plant was added to the garden.  Its kinda like a normal timestamp, but one we have more control over!